portfolio/src/main/webapp/images.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Food Journal</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <h1>Food Journal</h1>
+      <h2>June 2018-Present</h2>
+      <div class="image-container">
+        <a href="images/img1.JPG"><img src="images/img1.JPG"/></a>
```

Comment:
consider adding alt text for all these images.  it's always a good idea to keep accessibility in mind